*** Makefile.in	2007-10-21 20:06:45.468750000 +0200
--- Makefile.in2	2007-10-21 18:25:51.250000000 +0200
***************
*** 66,73 ****
  	oberror.cpp obiter.cpp obutil.cpp parsmart.cpp patty.cpp \
  	phmodel.cpp rand.cpp residue.cpp ring.cpp rotor.cpp \
  	rotamer.cpp tokenst.cpp typer.cpp transform.cpp \
! 	groupcontrib.cpp dlhandler_unix.cpp
! @BUILD_SHARED_TRUE@am__objects_1 = dlhandler_unix.lo
  am_libopenbabel_la_OBJECTS = atom.lo base.lo bitvec.lo bond.lo \
  	bondtyper.lo canon.lo chains.lo chiral.lo data.lo \
  	fingerprint.lo forcefield.lo generic.lo grid.lo kekulize.lo \
--- 66,73 ----
  	oberror.cpp obiter.cpp obutil.cpp parsmart.cpp patty.cpp \
  	phmodel.cpp rand.cpp residue.cpp ring.cpp rotor.cpp \
  	rotamer.cpp tokenst.cpp typer.cpp transform.cpp \
! 	groupcontrib.cpp dlhandler_win32.cpp
! @BUILD_SHARED_TRUE@am__objects_1 = dlhandler_win32.lo
  am_libopenbabel_la_OBJECTS = atom.lo base.lo bitvec.lo bond.lo \
  	bondtyper.lo canon.lo chains.lo chiral.lo data.lo \
  	fingerprint.lo forcefield.lo generic.lo grid.lo kekulize.lo \
***************
*** 248,254 ****
  INCLUDES = -I$(top_builddir)/include -I$(top_srcdir)/data/ -I$(top_srcdir)/include
  lib_LTLIBRARIES = libopenbabel.la
  @BUILD_SHARED_FALSE@dlhandler = 
! @BUILD_SHARED_TRUE@dlhandler = dlhandler_unix.cpp
  libopenbabel_la_SOURCES = \
  		atom.cpp base.cpp bitvec.cpp bond.cpp \
  		bondtyper.cpp canon.cpp \
--- 248,254 ----
  INCLUDES = -I$(top_builddir)/include -I$(top_srcdir)/data/ -I$(top_srcdir)/include
  lib_LTLIBRARIES = libopenbabel.la
  @BUILD_SHARED_FALSE@dlhandler = 
! @BUILD_SHARED_TRUE@dlhandler = dlhandler_win32.cpp
  libopenbabel_la_SOURCES = \
  		atom.cpp base.cpp bitvec.cpp bond.cpp \
  		bondtyper.cpp canon.cpp \
***************
*** 371,377 ****
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/chains.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/chiral.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/data.Plo@am__quote@
! @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/dlhandler_unix.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/fingerprint.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/forcefield.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/generic.Plo@am__quote@
--- 371,377 ----
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/chains.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/chiral.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/data.Plo@am__quote@
! @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/dlhandler_win32.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/fingerprint.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/forcefield.Plo@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/generic.Plo@am__quote@
