*** Makefile.in	2007-10-21 20:07:27.828125000 +0200
--- Makefile.in2	2007-10-21 20:14:45.828125000 +0200
***************
*** 32,38 ****
  build_triplet = @build@
  host_triplet = @host@
  bin_PROGRAMS = babel$(EXEEXT) obchiral$(EXEEXT) obenergy$(EXEEXT) \
! 	obfit$(EXEEXT) obgrep$(EXEEXT) obprop$(EXEEXT) \
  	obminimize$(EXEEXT) obrotate$(EXEEXT) obrotamer$(EXEEXT)
  subdir = tools
  DIST_COMMON = README $(srcdir)/Makefile.am $(srcdir)/Makefile.in
--- 32,38 ----
  build_triplet = @build@
  host_triplet = @host@
  bin_PROGRAMS = babel$(EXEEXT) obchiral$(EXEEXT) obenergy$(EXEEXT) \
! 	obfit$(EXEEXT) obprop$(EXEEXT) \
  	obminimize$(EXEEXT) obrotate$(EXEEXT) obrotamer$(EXEEXT)
  subdir = tools
  DIST_COMMON = README $(srcdir)/Makefile.am $(srcdir)/Makefile.in
***************
*** 59,67 ****
  am_obfit_OBJECTS = obfit.$(OBJEXT)
  obfit_OBJECTS = $(am_obfit_OBJECTS)
  obfit_DEPENDENCIES = ../src/libopenbabel.la
! am_obgrep_OBJECTS = obgrep.$(OBJEXT)
! obgrep_OBJECTS = $(am_obgrep_OBJECTS)
! obgrep_DEPENDENCIES = ../src/libopenbabel.la
  am_obminimize_OBJECTS = obminimize.$(OBJEXT)
  obminimize_OBJECTS = $(am_obminimize_OBJECTS)
  obminimize_DEPENDENCIES = ../src/libopenbabel.la
--- 59,67 ----
  am_obfit_OBJECTS = obfit.$(OBJEXT)
  obfit_OBJECTS = $(am_obfit_OBJECTS)
  obfit_DEPENDENCIES = ../src/libopenbabel.la
! #am_obgrep_OBJECTS = obgrep.$(OBJEXT)
! #obgrep_OBJECTS = $(am_obgrep_OBJECTS)
! #obgrep_DEPENDENCIES = ../src/libopenbabel.la
  am_obminimize_OBJECTS = obminimize.$(OBJEXT)
  obminimize_OBJECTS = $(am_obminimize_OBJECTS)
  obminimize_DEPENDENCIES = ../src/libopenbabel.la
***************
*** 87,96 ****
  	--mode=link $(CXXLD) $(AM_CXXFLAGS) $(CXXFLAGS) $(AM_LDFLAGS) \
  	$(LDFLAGS) -o $@
  SOURCES = $(babel_SOURCES) $(obchiral_SOURCES) $(obenergy_SOURCES) \
! 	$(obfit_SOURCES) $(obgrep_SOURCES) $(obminimize_SOURCES) \
  	$(obprop_SOURCES) $(obrotamer_SOURCES) $(obrotate_SOURCES)
  DIST_SOURCES = $(babel_SOURCES) $(obchiral_SOURCES) \
! 	$(obenergy_SOURCES) $(obfit_SOURCES) $(obgrep_SOURCES) \
  	$(obminimize_SOURCES) $(obprop_SOURCES) $(obrotamer_SOURCES) \
  	$(obrotate_SOURCES)
  ETAGS = etags
--- 87,96 ----
  	--mode=link $(CXXLD) $(AM_CXXFLAGS) $(CXXFLAGS) $(AM_LDFLAGS) \
  	$(LDFLAGS) -o $@
  SOURCES = $(babel_SOURCES) $(obchiral_SOURCES) $(obenergy_SOURCES) \
! 	$(obfit_SOURCES) $(obminimize_SOURCES) \
  	$(obprop_SOURCES) $(obrotamer_SOURCES) $(obrotate_SOURCES)
  DIST_SOURCES = $(babel_SOURCES) $(obchiral_SOURCES) \
! 	$(obenergy_SOURCES) $(obfit_SOURCES) \
  	$(obminimize_SOURCES) $(obprop_SOURCES) $(obrotamer_SOURCES) \
  	$(obrotate_SOURCES)
  ETAGS = etags
***************
*** 227,234 ****
  obenergy_LDADD = ../src/libopenbabel.la @LIBDL@
  obfit_SOURCES = obfit.cpp
  obfit_LDADD = ../src/libopenbabel.la @LIBDL@
! obgrep_SOURCES = obgrep.cpp
! obgrep_LDADD = ../src/libopenbabel.la @LIBDL@
  obminimize_SOURCES = obminimize.cpp
  obminimize_LDADD = ../src/libopenbabel.la @LIBDL@
  obprop_SOURCES = obprop.cpp
--- 227,234 ----
  obenergy_LDADD = ../src/libopenbabel.la @LIBDL@
  obfit_SOURCES = obfit.cpp
  obfit_LDADD = ../src/libopenbabel.la @LIBDL@
! #obgrep_SOURCES = obgrep.cpp
! #obgrep_LDADD = ../src/libopenbabel.la @LIBDL@
  obminimize_SOURCES = obminimize.cpp
  obminimize_LDADD = ../src/libopenbabel.la @LIBDL@
  obprop_SOURCES = obprop.cpp
***************
*** 311,319 ****
  obfit$(EXEEXT): $(obfit_OBJECTS) $(obfit_DEPENDENCIES) 
  	@rm -f obfit$(EXEEXT)
  	$(CXXLINK) $(obfit_OBJECTS) $(obfit_LDADD) $(LIBS)
! obgrep$(EXEEXT): $(obgrep_OBJECTS) $(obgrep_DEPENDENCIES) 
! 	@rm -f obgrep$(EXEEXT)
! 	$(CXXLINK) $(obgrep_OBJECTS) $(obgrep_LDADD) $(LIBS)
  obminimize$(EXEEXT): $(obminimize_OBJECTS) $(obminimize_DEPENDENCIES) 
  	@rm -f obminimize$(EXEEXT)
  	$(CXXLINK) $(obminimize_OBJECTS) $(obminimize_LDADD) $(LIBS)
--- 311,319 ----
  obfit$(EXEEXT): $(obfit_OBJECTS) $(obfit_DEPENDENCIES) 
  	@rm -f obfit$(EXEEXT)
  	$(CXXLINK) $(obfit_OBJECTS) $(obfit_LDADD) $(LIBS)
! #obgrep$(EXEEXT): $(obgrep_OBJECTS) $(obgrep_DEPENDENCIES) 
! #	@rm -f obgrep$(EXEEXT)
! #	$(CXXLINK) $(obgrep_OBJECTS) $(obgrep_LDADD) $(LIBS)
  obminimize$(EXEEXT): $(obminimize_OBJECTS) $(obminimize_DEPENDENCIES) 
  	@rm -f obminimize$(EXEEXT)
  	$(CXXLINK) $(obminimize_OBJECTS) $(obminimize_LDADD) $(LIBS)
***************
*** 337,343 ****
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obchiral.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obenergy.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obfit.Po@am__quote@
! @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obgrep.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obminimize.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obprop.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obrotamer.Po@am__quote@
--- 337,343 ----
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obchiral.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obenergy.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obfit.Po@am__quote@
! #@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obgrep.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obminimize.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obprop.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/obrotamer.Po@am__quote@
