--- src\plugins\imageformats\jpeg\jpeg.pro.orig	2007-08-07 15:22:00.000000000 +0200
+++ src\plugins\imageformats\jpeg\jpeg.pro	2007-09-13 21:50:54.934875000 +0200
@@ -8,8 +8,8 @@
            qjpeghandler.cpp
 
 contains(QT_CONFIG, system-jpeg) {
-        unix:LIBS += -ljpeg
-        win32:LIBS += libjpeg.lib
+        unix|win32-g++:LIBS += -ljpeg
+        win32-msvc*:LIBS += jpeg.lib
 }
 !contains(QT_CONFIG, system-jpeg) {
 	INCLUDEPATH += ../../../3rdparty/libjpeg
--- src\plugins\imageformats\tiff\tiff.pro.orig	2007-08-07 15:22:00.000000000 +0200
+++ src\plugins\imageformats\tiff\tiff.pro	2007-09-13 21:54:28.981750000 +0200
@@ -8,8 +8,8 @@
            qtiffhandler.cpp
 
 contains(QT_CONFIG, system-tiff) {
-        unix:LIBS += -ltiff
-        win32:LIBS += libtiff3.lib
+        unix|win32-g++:LIBS += -ltiff
+        win32-msvc*:LIBS += tiff.lib
 }
 !contains(QT_CONFIG, system-tiff) {
 	INCLUDEPATH += ../../../3rdparty/libtiff/libtiff
--- tools\qdbus\tools\qdbusviewer\qdbusviewer.rc.orig	2007-08-07 15:22:00.000000000 +0200
+++ tools\qdbus\tools\qdbusviewer\qdbusviewer.rc	2007-09-14 23:34:41.561250000 +0200
@@ -1 +1 @@
-IDI_ICON1		ICON		DISCARDABLE	"images/qdbusviewer.ico"
+IDI_ICON1		ICON		DISCARDABLE	"images/dbusviewer.ico"
